x-pack/test/functional/apps/ml/stack_management_jobs/manage_spaces.ts

```diff
@@ -179,7 +179,12 @@ export default function ({ getService }: FtrProviderContext) {
           await ml.navigation.navigateToStackManagementJobsListPage();
 
           // AD
-          await ml.jobTable.filterWithSearchString(testData.adJobId, 1, 'stackMgmtJobList');
+          await ml.stackManagementJobs.filterTableWithSearchString(
```

Comment:
These tests should be expanded to check trained models.
This can be done in a follow up PR